.github/workflows/build.yml

```diff
@@ -0,0 +1,165 @@
+name: cesium-native
+on: [push, pull_request]
+jobs:
+  QuickChecks:
+    name: "Quick Checks"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v3
+      - name: Check source formatting
+        run: |
+          npm install
+          npm run format -- --dry-run -Werror
+  Documentation:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Doxygen
+        run: |
+          sudo apt install -y doxygen
+      - name: Check out repository code
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+      - name: Generate Documentation
+        run: |
+          cmake -B build -S .
+          cmake --build build --target cesium-native-docs
+      - name: Publish Documentation Artifact
+        if: ${{ success() }}
+        uses: actions/upload-artifact@v3
+        with:
+          name: ReferenceDocumentation
+          path: build/doc/html
+  VS2019:
+    name: "Windows + VS2019"
+    runs-on: windows-2019
+    steps:
+      - name: Install nasm
+        run: |
+          choco install -y nasm
+      - name: Check out repository code
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+      - name: Compile Debug Configuration
+        run: |
+          cmake -B build -S .
+          cmake --build build --config Debug --parallel 4
+      - name: Test Debug Configuration
+        run: |
+          cd build
+          ctest -V
+      - name: Compile RelWithDebInfo Configuration
```

Comment:
Looks good here, but question for my sanity, did Travis ever build release? Because I didn't see it in the logs.

Reply:
No, Travis previously only built Debug.

Reply:
Fantastic improvement then, especially since tests are now run in release too.